charts/smile/README.md

```diff
@@ -0,0 +1,160 @@
+## SMILE (Social Media Intelligence and Learning Environment)
+![logo](www/public/img/logo/square2.png)
+
+### For users or potential users:
+- [SMILE](https://smile.smm.ncsa.illinois.edu)
+- Watch the **[Overview video](https://www.youtube.com/watch?v=ySb-8TwJ0Ds)**
+- Contact us the **[SRTI lab](https://srtilab.techservices.illinois.edu/about/)** if you have any question: <a href="mailto:srti-lab@illinois.edu">srti-lab@illinois.edu</a>
+- [SMILE Playground](https://smm-tester.ncsa.illinois.edu) coming soon...
+
+### For developers:
+#### Structure of SMILE:
+![Architecture](./www/public/img/SMILE_architecture.png)
+
+## TL;DR;
```

Comment:
Could we just take out the outdated information? 
e.g. mailing list, architecture graph. 
Since this will be published to bitnami and changing that would be hard. I have separate tasks for cleaning out the readme everywhere.

Reply:
Sounds good. Removed those lines

Reply:
for the mailing list, could you change to smm@lists.illinois.edu, also remove anything related to srti lab? That information was way outdated...

Reply:
oh didn't know about that. Changed.